nextmind-stretch-server/app.py

```diff
@@ -4,42 +4,84 @@
 from starlette.responses import JSONResponse
 from starlette.routing import Route
 
-path_to_commands = "/home/hello-robot/Chatbot/moveGrip.py"
+path_to_commands = "/home/hello-robot/Chatbot"
```

Comment:
consider creating a function, which accepts a command, e.g. moveGrip

and then calls:
os.system(f"python {path_to_commands}/{command}.py {params}) so you can reuse the same function.

Reply:
I'd uppercase this as a constant, per PEP8